src/i18n/en-US/modelPlanTaskList.ts

```diff
@@ -97,9 +97,9 @@ const modelPlanTaskList = {
       path: 'payment'
     },
     itSolutions: {
-      heading: 'IT solutions and implementation status',
+      heading: 'Operational solutions and implementation status',
       team:
-        'Choose the IT solutions your model model will utilize. Many items in this section are populated based on responses to questions answered in previous sections.',
+        'Choose the solutions your model model will utilize. Many items in this section are populated based on responses to questions answered in previous sections.',
```

Comment:
Copy is not correct. There is an extra "model"

> Choose the solutions your **model model** will util....

Reply:
Lol that has always been there, I didn't touch that code.  Nice catch though